notebooks/Working with MONAI Auto3DSeg using MONAI Cloud API.ipynb

```diff
@@ -24,35 +24,121 @@
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "import json\n",
+    "import os\n",
     "import requests\n",
-    "\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Helper functions for running jobs\n",
+    "def wait_for_job(endpoint, headers, timeout):\n",
+    "    start_time = time.time()\n",
+    "    response = requests.get(endpoint, headers=headers)\n",
+    "    assert response.status_code == 200, f\"Failed to get job status, got {response.json()}.\"\n",
+    "    status = response.json()[\"status\"].title()\n",
+    "    print(\"Waiting for job to complete...\")\n",
+    "    print(status, end=\"\", flush=True)\n",
+    "    while True:\n",
+    "        if status not in [\"Pending\", \"Running\"]:\n",
+    "            assert status == \"Done\", f\"Job failed with status: {status}\"\n",
+    "            break\n",
+    "        time.sleep(5)\n",
+    "        response = requests.get(endpoint, headers=headers)\n",
+    "        assert response.status_code == 200, f\"Failed to get job status, got {response.json()}.\"\n",
+    "        status_new = response.json()[\"status\"].title()\n",
+    "        if status_new != status:\n",
+    "            status = status_new\n",
+    "            print(f\"\\n{status}\", end=\"\", flush=True)\n",
+    "        else:\n",
+    "            print(\".\", end=\"\", flush=True)\n",
+    "        if time.time() - start_time > timeout:\n",
+    "            assert False, f\"Job timeout after {timeout} seconds.\"\n",
+    "    print(\"\\nJob completed successfully!\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### <<< Following Parameters Need To Be Provided >>>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
     "# API Endpoint and Credentials\n",
     "monai_cloud_api = \"<MONAI Cloud API URL>\"\n",
-    "api_url = f\"{monai_cloud_api}/api/v1\"\n",
     "ngc_api_key = \"<NGC API Key>\"\n",
     "\n",
-    "# NGC UID \n",
-    "response = requests.get(f\"{api_url}/login/{ngc_api_key}\")\n",
+    "# Object Storage URL and Credentials\n",
+    "train_data_url = \"<container url for the training dataset>\"      # Training dataset container URL\n",
+    "inference_data_url = \"<container url for the inference dataset>\" # Inference dataset container URL\n",
+    "storage_client_id = \"<object storage ID>\"                        # Object storage username/id\n",
+    "storage_client_secret = \"<object storage secret>\"                # Object storage password/secret\n",
+    "\n",
+    "# Time out for executing the training and inference job\n",
+    "timeout = 600 # seconds"
```

Comment:
Also could move this closer to the job code block

Reply:
Hi @zephyrie , since we'd like to use papermill to run notebooks in CI, an limitation is that all parameters to be changed should be in the same cell. That's why we put both secret and timeout (for training) variables in the same place.
Do you think if we can have other ways to make this cell more clear?

Reply:
Update: I think we can use a small timeout value directly thus no need to override for CI. Will update it.